carddav/server.go

```diff
@@ -262,7 +262,12 @@ func (b *backend) HeadGet(w http.ResponseWriter, r *http.Request) error {
 	}
 
 	w.Header().Set("Content-Type", vcard.MIMEType)
-	// TODO: set ETag, Last-Modified
+	if ao.ETag != "" {
+		w.Header().Set("ETag", ao.ETag)
+	}
+	if !ao.ModTime.IsZero() {
+		w.Header().Set("Last-Modified", ao.ModTime.Format(http.TimeFormat))
```

Comment:
From the `net/http` docs:

> The time being formatted must be in UTC for Format to generate the correct format. 

So need to add an `.UTC()` in there, in case the backend returns a non-UTC date.